core/ts/lib/cryptoUtils.ts

```diff
@@ -0,0 +1,143 @@
+import * as assert from "assert";
+
+interface TreeDepths {
```

Comment:
maybe worth separating interfaces to their own file too (interfaces.ts or types.ts could be a good name for that)

Reply:
I would like to address it in the following PR. The main goal of this PR is to improve the visibility of the core components of the `MaciState` and `Poll` classes in our codebase.

Reply:
As long as you address it as some point 👍